src/redux/books/bookReducer.js

```diff
@@ -1,13 +1,48 @@
-import * as Actions from './bookAction';
+import { v4 as uuidv4 } from 'uuid';
+import * as actions from '../../action/bookAction';
 
-const bookReducer = (state = {}, action = {}) => {
+export const initialBooks = [
+  {
+    id: uuidv4(),
+    title: 'Book One',
+    author: 'author',
+    genre: 'Fiction',
+  },
+  {
+    id: uuidv4(),
+    title: 'Book Two',
+    author: 'author',
+    genre: 'Economy',
+  },
+  {
+    id: uuidv4(),
+    title: 'Book Three',
+    author: 'author',
+    genre: 'Documentation',
+  },
+];
+
+const books = (state = initialBooks, action) => {
   switch (action.type) {
-    case Actions.ADDED_BOOK:
-    case Actions.REMOVED_BOOK:
-    case Actions.SHOWED_BOOK:
+    case actions.ADDED_BOOK: {
```

Comment:
- [ ] You did not define actions yet. Actions are defined with type and payload attributes. Payload is optional and it is passed as an argument in the action method. Please see the below general example of how to simply define a redux action. Kindly make changes here. :pray: In that case, you don't need to say `case actions.ADDED_BOOK` but instead you can say `case: ADDED_BOOK`. In addition, the action type naming is better if you say `ADD_BOOK` and `REMOVE_BOOK` as it is an action and not a past event. 

```
export const removeBook = (payload) => ({
  type: ACTION_TYPE,
  payload,
});
```

Reply:
Noted

Reply:
This issue is well addressed! :+1:

Reply:
No problem